src/mlpack/methods/ann/layer/softshrink.hpp

```diff
@@ -0,0 +1,224 @@
+/**
+ * @file softshrink.hpp
+ * @author Lakshya Ojha
+ *
+ * The soft shrinkage has thresholds proportional to the noise level given by 
```

Comment:
Could you use same name as the class i.e. soft shrink function. Thanks.

Reply:
Thanks for noticing changed the same.
Thanks.